FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Exceptions/ErrorProcessors/HealthCheckErrorProcessor.cs

```diff
@@ -0,0 +1,27 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Microsoft.Health.Fhir.Synapse.Common.Metrics;
+
+namespace Microsoft.Health.Fhir.Synapse.Core.Exceptions.ErrorProcessors
+{
+    public class HealthCheckErrorProcessor
+    {
+        private IMetricsLogger _metricsLogger;
+
+        public HealthCheckErrorProcessor(IMetricsLogger metricsLogger)
+        {
+            _metricsLogger = EnsureArg.IsNotNull(metricsLogger, nameof(metricsLogger));
+        }
+
+        public void Process(Exception ex, string message = "")
```

Comment:
string message? I feel message+=ex.Message is not a good pactice here. Can we reuse this message just like other loggers ?

Reply:
Updated it.